apps/SageAccountWeb/src/components/AccountSettings.tsx

```diff
@@ -236,7 +236,9 @@ export const AccountSettings = () => {
                 </Typography>
                 <Typography variant={'body1'}>
                   This information is reused across all{' '}
-                  <RouterLink to="/sageresources">Sage products.</RouterLink>
+                  <RouterLink to="/sageresources">
+                    Synapse resources.
+                  </RouterLink>
```

Comment:
"Synapse resources" to refer to these apps feels awkward. I would expect "Synapse resources" to mean "content in Synapse"

Reply:
Yes, we refer to them as products and resources (see Account Created Page above).  I'd be happy to change to products, but I just want them to be consistent.